_posts/2022-06-20-github-actions-orchestrator.md

```diff
@@ -43,7 +43,7 @@ This tutorial assumes that you have:
 * [Git](https://git-scm.com/) installed
 * a [GitHub](https://github.com/) account
 * [Docker](https://www.docker.com/) installed and running
-* [Remote ZenML Server](https://docs.zenml.io/getting-started/deploying-zenml#deploying-zenml-in-the-cloud-remote-deployment-of-the-http-server-and-database) A Remote Deployment of the ZenML HTTP server and Database
+* [Remote ZenML Server](https://docs.zenml.io/platform-guide/set-up-your-mlops-platform/deploy-zenml) A Remote Deployment of the ZenML HTTP server and Database
```

Comment:
should we add a big "DEPRECATED/REMOVED" warning somewhere on top of this post since the orchestrator is no longer part of ZenML?

Reply:
Or maybe remove this post altogether?

Reply:
It's within the top-15-most-viewed posts on our blog, so maybe just mention that it's been removed from zenml core but don't delete the post. (Any removals should probably be checked against our plausible stats anyway).

Reply:
I agree with @strickvl. This is why I think any possible removals should be discussed and executed in a separate PR/ticket.

Reply:
Ok then a warning at the top might be nice